InventoryInStorePickupShippingApi/Model/IsInStorePickupDeliveryAvailableForCart.php

```diff
@@ -0,0 +1,123 @@
+<?php
+declare(strict_types=1);
+
+namespace Magento\InventoryInStorePickupShippingApi\Model;
+
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Quote\Api\CartRepositoryInterface;
+use Magento\Quote\Api\Data\CartInterface;
+use Magento\Quote\Api\Data\EstimateAddressInterface;
+use Magento\Quote\Api\Data\EstimateAddressInterfaceFactory;
+use Magento\Quote\Api\Data\ShippingAssignmentInterface;
+use Magento\Quote\Api\ShippingMethodManagementInterface;
+
+/**
+ * Check if In-Store Pickup delivery method is applicable for a cart by cartId.
+ *
+ * @api
+ */
+class IsInStorePickupDeliveryAvailableForCart
```

Comment:
Could you move implementation into **InventoryInStorePickupShipping** and here only leave an interface.

Reply:
I'm not sure that we need interface here. API model looks good as for me

Reply:
Well, I will create an interface while it is not too late.

Reply:
Done.